onnxruntime/test/perftest/ort_test_session.cc

```diff
@@ -5,6 +5,7 @@
 #include <assert.h>
 #include "providers.h"
 #include "TestCase.h"
+#include "Tracy.hpp"
```

Comment:
Should this be inside a `#ifdef`? I wouldn't expect the Tracy.hpp header to exist/be found in most cases.

Reply:
See https://github.com/microsoft/onnxruntime/pull/10830#discussion_r826785513